src/test/java/com/fasterxml/jackson/core/read/NonStandardNumberParsingTest.java

```diff
@@ -243,6 +243,20 @@ void leadingDotInNegativeDecimalAllowedReader() throws Exception {
         _testLeadingDotInNegativeDecimalAllowed(jsonFactory(), MODE_READER_THROTTLED);
     }
 
+    // https://github.com/FasterXML/jackson-databind/issues/4694
```

Comment:
Can remove, added concurrently as `NumberParsing4694Test.java` under `src/test/java/.../failing`, to be merged in `NumberParsingTest` once fixed.